ui/src/publication/log/publication-log.tsx

```diff
@@ -77,14 +79,17 @@ const PublicationLog: React.FC = () => {
 
         setIsLoading(true);
 
+        const currentFetchId = ++fetchId;
```

Comment:
Additionally another minihuomio: 🤔 Meillä on näitä niin vähän, että lienee ok. 

💭 Tottunut kuitenkin itse vähän välttämään erityisesti tuota pre-incrementtiä, sillä varsinkin junnummille on syystä tai toisesta usein vaikeaa ymmärtää ++jotain vs jotain++ ero. Indeksoinneissa tällaiset aiheuttavat myös herkästi bugeja jotka eivät välttämättä jää kiinni katselmoinneissa, koska koodin lukijalle ei ole välttämättä selvää mitä alkuperäinen kirjoittaja yritti tehdä (esim. jääkö fetchId = 0 tässä tarkoituksella käyttämättä? Näin tulkitsen käyvän, mutten tiedä miksi näin tehdään, ja tämän huomaaminen edellyttää että tietää eron pre/post-incrementin välillä assignauksessa). 

Siispä tullut totuttua yksinkertaisemmin ymmärrettävämpään tapaan, joka myös välttää bugeja niin logiikassa kuin ymmärryksessäkin:
```
fetchId += 1;
const currentFetchId = fetchId;

tai

const currentFetchId = fetchId;
fetchId += 1;
```

Tosin oma tottumus johtuu varmaan seuraavista:
* Python ei tue pre-/post-increment/decrement operaattoria lainkaan, edellyttää käytön tuolla tavoin lauseena eri rivillä
* Go tukee vain lauseena eli fetchId++ pitäisi olla omana rivinään (ei voi asettaa suoraan muuttujaan eikä pre-incrementtiä ole lainkaan). Go tukee tietysti vastaavasti myös tuota fetchId += 1 formaattia incrementin sijaan
* Rusti ei myöskään tue pre/post-increment/decrementtiä lainkaan ja edellyttää myös käytön lauseena omalla rivillä eli fetchId += 1